praw/reddit.py

```diff
@@ -107,7 +107,7 @@ def __init__(
             used to initialize the requestor (default: None).
 
         Additional keyword arguments will be used to initialize the
-        :class`.Config` object. This can be used to specify configuration
```

Comment:
Can we make it so the Config object links properly?

Reply:
Changed.

Reply:
Thanks!